README.md

```diff
@@ -72,7 +72,7 @@ This scenario applies if:
 import Ember from 'ember';
 import pagedArray from 'ember-cli-pagination/computed/paged-array';
 
-Ember.ArrayController.extend({
+export default Ember.ArrayController.extend({
```

Comment:
@Robdel12 probably a good idea. Need to confirm I'm not doing anything that relies on ArrayController